.github/workflows/lokalise-automerger.yml

```
@@ -0,0 +1,71 @@
+name: Lokalise PR validation, auto approve and merge
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+    paths:
+      - 'src/locale/*.json'
```

> Ideally, this action will only run when either the src/locale/nl.json file or the src/locale/en.json has been changed.

In that case, shouldn't the paths be the following?

```suggestion
    paths:
      - 'src/locale/en.json'
      - 'src/locale/nl.json'
```

---

No, this will effectively do the same. If BESIDES these two files OTHER files are changed as well, the action will still run. WHat I'm trying to achieve is that this action will run EXCLUSIVELY when ONLY these two files are in the changeset.

---

Ah ok, maybe something like https://stackoverflow.com/questions/59288971/retrieving-list-of-modified-files-in-github- might point you in the right direction, but I'm not sure if it's worth the effort

---

Yes, with the use of some other actions I'll be able to get it to work, but all of those solutions will require the action to be started up. If it would be possible just by configuring the paths at the top of the action, in that case it won't even be started by Github, which is the speedup I'm looking for.
But yes, it's a nitpick, so currently not worth the bother.